<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSeg</class>
 <widget class="QDialog" name="DialogSeg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Segunda</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
